RadialGM.pro

```diff
@@ -20,14 +20,14 @@ QMAKE_TARGET_PRODUCT = RadialGM IDE
 QMAKE_TARGET_DESCRIPTION = ENIGMA Development Environment
 QMAKE_TARGET_COPYRIGHT = "Copyright \\251 2007-2018 ENIGMA Dev Team"
 
-# Uncomment if you want QPlainTextEdit used in place of QScintilla
-CONFIG += rgm_disable_syntaxhighlight
+# Comment out if you want QPlainTextEdit used in place of QScintilla
```

Comment:
The comment shouldn't invite people to take an action that's already be taken. I know it was like this before, but it's confusing.

Reply:
Alright: 08d1f0636feddaffabe67d0dc1329e79dfdd7941